src/GafferSceneUI/SceneGadget.cpp

```diff
@@ -159,7 +159,7 @@ void SceneGadget::setPaused( bool paused )
 	}
 	else if( m_controller.updateRequired() )
 	{
-		requestRender();
+		dirty( DirtyType::Bound );
```

Comment:
Is it `Bound` here as the scene bounds may have changed whilst we were paused?

Reply:
I was more thinking that we'll end up updating the renderer, which will affect the bound. But actually `bound()` doesn't call `updateRenderer()` anyway, and relies on `doRenderLayer()` for that. It's all a bit moot since `bound()` changes dynamically as the scene is updated in async fashion, so it wouldn't work to try to use a SceneGadget in a layout. I think it can be argued either way, but I'm fairly happy with the more conservative dirtying of Bound, even if the actual changes are asynchronous and impossible to track.